provision/acc_provision/acc_provision.py

```diff
@@ -1611,12 +1599,12 @@ def generate_kube_yaml(args, config, operator_output, operator_tar, operator_cr_
             network_operator_spec_output = network_operator_spec_template.render(config=config)
             network_operator_openshift_spec_output = network_operator_openshift_spec_template.render(config=config)
 
-            network_operator_CR_template = get_jinja_template('cni/aci/' + ver + '/aci-installer-cr.yaml')
+            network_operator_CR_template = get_jinja_template(ACI_CNI_VERSIONS_PATH + '/aci-installer-cr.yaml')
             base64_encoded_cko_aci_spec = base64.b64encode(new_deployment_file.encode('ascii')).decode('ascii')
             netopConfig["aci_config"]["operator_version"] = netop_version
             netopConfig["aci_config"]["cni_flavor_version"] = config["registry"]["version"]
             netopConfig["aci_config"]["base64_encoded_cko_aci_spec"] = base64_encoded_cko_aci_spec
-            network_operator_platform_CR_template = get_jinja_template('cni/aci/' + ver + '/platform-installer-cr.yaml')
+            network_operator_platform_CR_template = get_jinja_template('cko/' + netop_version + '/platform-installer-cr.yaml')
```

Comment:
Why are we not following the pattern of computing the path once and assigning it to a variable for CKO as well?

Reply:
so I thought of maintaining the previous implementation that existed in the code. Here - https://github.com/noironetworks/acc-provision/pull/854/files/0bf5d4ca13bcfbb798758310965a902449369425#diff-d18edd87a3bcaa21be667d28554472484dbc49b6c8839bfddec4c68545f9c559R1481 . Should we change this? and will change everywhere cko/ + netop_version is used?

Reply:
I think we should...

Reply:
sure working on both comments